CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
+        self.assertEqual(7, Calculator.run_operation(7, 10, '%'))
```

Comment:
Test case could use more variety in number input to check for additional errors that may occur.

Reply:
Agreed, we could implement some error handling for invalid inputs for first and second numbers